index.html

```diff
@@ -1,11 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <link rel="stylesheet" href="styles.css">
     <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-  
+    <h1 id = "title">✨Self Care Center✨</h1>
+
+    <section class = >
+      <h2 id = "sub-title">Which type of message?</h2>
+    </section>
+
+    <form class = "radio-box">
+      <div>
+        <input type ="radio" id ="choice1" name ="radio-button" value = "shallow-thought" checked>
+        <label for ="choice1"><em>Shallow Thoughts</em></label>
```

Comment:
Don't use style elements like < em > . Instead, you should have something like so in your css:

.text-italic {
  font-style: italic;
}

This way whenever you need italic text you just add `class="text-italic" and it will apply it. This is usally a good design pattern. Having singular CSS rules like above allow for you to go down into your html and just add that property to a class when you need it

Reply:
Understood. I was trying to write semantic html also. I will make sure to add the CSS rule, and remove the <em> elements.